src/load/load.types.ts

```diff
@@ -1,7 +1,22 @@
-export interface ImageMetadata {
+export interface TiffImageMetadata {
   tiff: {
     fields: Map<number, unknown>;
     tags: Record<string, unknown>;
   };
   exif: Record<string, unknown>;
 }
+
+export interface PngImageMetadata {
+  resolution: Resolution;
+}
+
+export type Resolution =
+  | { x: number; y: number; unit: null }
+  | { x: number; y: number }
```

Comment:
Why are both `null` and "not set" possible for the resolution unit?

Reply:
Yeah, I can see now that it's confusing. I thought that if the values are in meters, there's no need for specifying units. And if the unit is unknown, then it's `null`. I will add "meter" as one of the units